.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+name: Python package
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Python Code Quality and Lint
+      uses: ricardochaves/python-lint@v1.1.0
+      with:
+        python-root-list: pyvistaqt
+        use-pylint: false
+        use-pycodestyle: false
+        use-flake8: false
+        use-black: true
+        use-mypy: false
+        use-isort: false
+        extra-pylint-options: --disable=F0401
+        extra-pycodestyle-options: --max-line-length=88 --ignore="W503"
+        extra-mypy-options: --no-implicit-optional
+        extra-flake8-options: --max-line-length=88
```

Comment:
Since `pylint`, `flake8`, `mypy` and `pycodestyle` are disabled, I don't think the extra options are necessary. Overall, I would just keep it with the defaults:

```py
      with:
        python-root-list: pyvistaqt
        use-black: true
```

Reply:
Looking at https://github.com/ricardochaves/python-lint/blob/master/action.yml, all of that is enabled by default so you're right, the following seems necessary:

```py
        use-pylint: false
        use-pycodestyle: false
        use-flake8: false
        use-mypy: false
        use-isort: false
```

Reply:
Thanks for your comment. I think also pylint, pycodestyle, flake8, mypy, isort is also useful. How about set enable these option too?

Reply:
This comment has not been resolved, @GuillaumeFavelier what do you prefer?

I agree they should be enabled, or their options removed.

Reply:
@larsoner Thanks I will set it enabled.

Reply:
I think this time. It it good to add only black option and remove extra option.

Reply:
Enable all option has a lot of error. I think it is not a good idea to enable all option. I will remove extra option.